assets/js/components/Popup.js

```diff
@@ -318,8 +342,16 @@ module.exports = class Popup {
     // Changing the state depending on the other properties of the object.
     if (this.isOpen) {
       this.positionPopupHitBox(e);
+
+      this.popupHitBox.addEventListener('blur', () => {
```

Comment:
This `blur` event listener callback  could ideally `removeEventListener` itself to prevent multiple event listener registrations when the same popup hit box is opened/closed/opened multiple times on a page. In reality I can't see that happening more than a handful of times, so it shouldn't present any performance issues. Because of this extremely low risk, I don't think any action is required.